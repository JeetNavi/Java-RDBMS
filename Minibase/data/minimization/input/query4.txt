Q(x) :- P(x,a), P(a,b), P(y,b), T(b,b,a)
