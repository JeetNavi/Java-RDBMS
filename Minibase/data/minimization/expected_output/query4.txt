Q(x) :- P(x, a), P(a, b), T(b, b, a)